Gemfile

```diff
@@ -1,3 +0,0 @@
-source 'https://rubygems.org'
-
-gem 'nokogiri'
```

Comment:
Recent versions of Redmine depend on `nokogiri` too, so this dependency is not needed (moreover, it caused conflict).